src/server/controllers/user.js

```diff
@@ -1,39 +1,43 @@
-import bcrypt from 'bcrypt';
-import jwt from 'jsonwebtoken';
-import { PrismaClient } from '@prisma/client'
-const prisma = new PrismaClient();
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
 
-const jwtSecret = 'mysecret';
+import { registerUserDB, findUser } from '../domain/user.js';
+const jwtSecret = "mysecret";
 
 const register = async (req, res) => {
     const { username, password } = req.body;
 
-    const createdUser = null;
-
-    res.json({ data: createdUser });
+    if (!password || !username) {
+        return res.status(409).json({error: "Please enter username or password"});
```

Comment:
This would actually be 400, not 409. 409 is for conflicts, for example, if they tried to register with a username that already is in-use

Reply:
Ah, good catch.